elyra/pipeline/processor_local.py

```
@@ -144,23 +144,24 @@ async def process(self, operation: Operation, filepath: str):
         pass
 
 
-class NotebookOperationProcessor(OperationProcessor):
-    _operation_name = 'execute-notebook-node'
+class FileOperationProcessor(OperationProcessor):
 
-    @property
-    def operation_name(self) -> str:
-        return self._operation_name
+    @abstractmethod
+    def _create_execute_command(self, filepath: str, cdw: str) -> str:
+        pass
```

We should probably raise `NotImplementedError` here rather than ignore that.